### PR TITLE
Set max_identifier_length to the value specified in the Redshift documentation

### LINKED ISSUE #96: Maximum identifier length is incorrect
Getting an error when trying to create a table with a name longer than 63 characters, even though the Redshift docs state that 127 characters is the limit:

http://docs.aws.amazon.com/redshift/latest/dg/r_names.html

### PR DESCRIPTION
**Pull request by scribu**
## Todos
- [x] MIT compatible
- [x] Tests
- [x] Updated CHANGES.rst

Fixes #96

### CODE REVIEW

**Comment by graingert** [comment]
Can you include a test for this? Eg create a table with a 127 char table name.

**Comment by scribu** [comment]
Sure. Can you point me to any existing tests that actually create a table in Redshift?

I saw `test_ddl_compiler.py`, but that only seems to test the generated SQL string.

**Comment by graingert** [comment]
@scribu can you create a failing test in test_ddl_compiler for the existing code in master, that this then fixes?

If so that would be good enough.

**Comment by graingert** [comment]
@scribu if you create a model with a huge char it should work.

```
class HugeTableName(Base):
    huge_a = sa.Column(name='a' * 127)
```

you should be able to create a test that uses this model, saves it and reads it out again

See https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/blob/master/tests/rs_sqla_test_utils/models.py

File: tests/test_long_tablename.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from rs_sqla_test_utils import models
+
+
+def test_long_tablename(redshift_session):
+    session = redshift_session
+    examples = [models.LongTablename(metric=i) for i in range(5)]
+    session.add_all(examples)
```
Comment (graingert) [inline tests/test_long_tablename.py]: You don't need the flush here

File: tests/rs_sqla_test_utils/models.py [hunk 2]
```diff
@@ -190,3 +190,9 @@ class PostfetchExample(Base):
 
     id = sa.Column(sa.Integer, primary_key=True, info={'identity': (0, 1)})
     some_int = sa.Column(sa.Integer)
+
+
+class LongTablename(Base):
+    __tablename__ = 'a' * 127
+
+    metric = sa.Column(sa.Integer, primary_key=True)
```
Comment (graingert) [inline tests/rs_sqla_test_utils/models.py]: Can we set metric to have a 127 char column name?

**Comment by scribu** [comment]
It looks like Travis is skipping `test_long_tablename.py` for some reason.

**Comment by graingert** [comment]
@scribu it's skipped because it needs to use the redshift credentials. You should create a plain SQL generation version too

**Comment by scribu** [comment]
A plain SQL generation test does not seem useful, since there isn't anything specific to Redshift about that. If _you_ feel like writing one, please do. In other words, I feel like I've checked off the "Tests" checkbox.

Not sure what the "MIT Compatible" and "Documentation" checkboxes mean for this specific PR.

**Comment by graingert** [comment]
Just check it, if this code is compatible with the MIT license. Eg can I merge it and release it under the existing license in this repo.

**Comment by graingert** [comment]
can you rebase this please, I've reverted the changes to sequences support.

**Comment by scribu** [comment]
Rebased.

**Comment by graingert** [comment]
can you check the "MIT compatible" checkbox to confirm that you want to license this code under the LICENSE that exists in this repo

**Comment by scribu** [comment]
Sure. Done.

**Pull request merged**